src/components/Species/ProblemTooltip.tsx

```diff
@@ -62,7 +64,10 @@ export default function ProblemTooltip({
   };
 
   const acceptFix = async (problemId: number) => {
-    await acceptProblemSuggestion(problemId);
+    const response = await acceptProblemSuggestion(problemId);
+    if (!(response && response.requestSucceeded)) {
```

Comment:
can just be `if (!response.requestSucceeded) ` since by contract you will get a response